tmt-app/src/Components/MajorCard.js

```diff
@@ -1,6 +1,142 @@
 import React from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Stack,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
 
 function MajorCard() {
-  return <div>MajorCard</div>;
+  const dummyData = [
+    {
+      id: 0,
+      name: "Education",
+      desc: "testing",
+      icon: "emotional_intelligence",
+    },
+    {
+      id: 1,
+      name: "Economics and Business",
+      desc: "test fdasf afas fsadf dsf dsf edsf ds fsdaf sd f asfsd fea fgdsffdsf dfasdf ds fsdafsdf fgsdaf ased fasd few gasef sWEAFGASDFAS FDFSA DFA FSADF EAFSDASD FGREAASDFG EF GRasdfga gadfaefadf asf ",
+      icon: "chemistry",
+    },
+  ];
+  return (
+    <Stack
+      spacing={3}
+      sx={{
+        width: "100%",
+        "&::after": {
+          content: "''",
+          width: "90%",
+          margin: "3rem auto",
+          border: "1px solid black",
+        },
+      }}
+    >
+      {dummyData.map((item) => {
+        return (
+          <Card
+            key={item.id}
```

Comment:
This should be a separate component- MajorCard

Reply:
Are you referring to the dummyData?

Reply:
I believe Michael is suggesting you should abstract this out into its own component

Reply:
@michachan thoughts on this? Ticket for later to clean up?